README.md

```diff
@@ -213,6 +214,18 @@ may be used for some string type configuration entries as detailed below.
 
 ### Configuration File Options
 
+#### __Extensions.Enabled__
+
+_Type: Boolean_  
```

Comment:
Please add this to the various waagent.conf versions in the tree.  I think people are more likely to read that file than they are the web site.

Reply:
Good call, updated